File: probs.py
from playwright.sync_api import sync_playwright

def fetch_div_content_with_playwright(url):
    with sync_playwright() as p:
        browser = p.chromium.launch(headless=True)
        context = browser.new_context()
        page = context.new_page()
        
        # Navegar a la URL
        page.goto(url)
        
        # Esperar a que el elemento div.pct esté presente en el DOM
        page.wait_for_selector('div.pct')
        
        # Obtener el contenido del div.pct
        div_content = page.query_selector('div.pct').inner_text()
        
        print(div_content)  # Imprime el contenido del div.pct
        
        browser.close()

url = 'https://www.futbolfantasy.com/jugadores/jan-oblak'
fetch_div_content_with_playwright(url)

File: probs.js
const puppeteer = require('puppeteer');

async function getPlayerProbs(url) {
    const browser = await puppeteer.launch({
        headless: true,
        args: ['--no-sandbox', '--disable-setuid-sandbox', '--disable-blink-features=AutomationControlled']
    });
    const page = await browser.newPage();
    await page.setViewport({ width: 1280, height: 800 });
    await page.setUserAgent('Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/91.0.4472.124 Safari/537.36');
    await page.goto(url);
    await page.waitForSelector('.pct');

    const divContent = await page.$eval('.pct', div => div.innerText);
    await browser.close();

    return divContent;
}

module.exports = getPlayerProbs;
